docs/modules/ROOT/pages/cilium/network-policies.adoc

```diff
@@ -0,0 +1,49 @@
+= Cilium Network Policies
+
+Cilium provides standard network policies for you cluster via the NetworkPolicy resource as well as extended functions via CiliumNetworkPolicy as CustomRessourceDefinition.
+
+You can read about network policies here via https://docs.openshift.com/container-platform/4.16/networking/network_security/network_policy/about-network-policy.html[upstream documentation].
+
+You can find examples on how to use the extended funcitonality of cilium network policies https://docs.cilium.io/en/latest/network/kubernetes/policy/#ciliumnetworkpolicy[here].
+
+== Default Network Policiecs
+
+Most clusters have two default network policies applied. You can list them in your namespace using:
+[source,shell]
+--
+oc get networkpolicy
+--
+
+
+`allow-from-other-namespaces`:: Allows incoming traffic from the monitoring stack, cluster ingresses, and the host network.
+
+`allow-from-same-namespace`:: Allows unrestricted traffic within the same namespace.
+
+== Interaction of Network Policies and Cilium Cluster Mesh
+
+When using Cilium Cluster Mesh, it's important to document how network policies interact across clusters, as the behavior may be unexpected.
+
+The default allow-from-same-namespace policy applies within an OpenShift namespace. While namespaces are unique within a single cluster, this constraint does not extend to a meshed cluster. Since the policy matches based on namespace name, it applies to any namespace with the same name across clusters. For example, if you have a `test` namespace on both `cluster1` and `cluster2`, pods in these namespaces can communicate freely across clusters. This show that, generally, network policies treat all clusters as a single unit, though they remain cluster-aware.
```

Comment:
```suggestion
The default allow-from-same-namespace policy applies within an OpenShift namespace.
While namespaces are unique within a single cluster, this constraint does not extend to a meshed cluster.
Since the policy matches based on namespace name, it applies to any namespace with the same name across clusters. For example, if you have a `test` namespace on both `cluster1` and `cluster2`, pods in these namespaces can communicate freely across clusters.
This show that, generally, network policies treat all clusters as a single unit, though they remain cluster-aware.
```